west.yml

```diff
@@ -321,7 +321,7 @@ manifest:
       revision: b84bd7314a239f818e78f6927f5673247816df53
       path: modules/bsim_hw_models/nrf_hw_models
     - name: nrf_wifi
-      revision: 0cd3bb2291a7dd22f4cdb1a4cd935a213c2bbfab
+      revision: pull/32/head
```

Comment:
`662ed74dce955e6c243f91c45a66768f5971e3cb`

Reply:
Can we move forward with this? It is blocking CI.

Reply:
just updated the commit with the final sha ...

Reply:
Thanks, it was a holiday and couldn't take care of this.